bitnami/kube-prometheus/templates/exporters/core-dns/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.coreDns.enabled }}
```

Comment:
why are we adding `core-dns` if we are not installing it? it is using a selector base on the `kube-prometheus` release like the following but the chart is not installing it. 

```
app.kubernetes.io/component: {{ template "kube-prometheus.fullname" . }}-coredns
```

Reply:
I am not sure if I am understand. The core-dns Service (and ServiceMonitor) are installed (coreDns.enabled default is true in values). Selector is "k8s-app: kube-dns" - on my test cluster I can see both components installed, as well as Prometheus targets UP. Could you give me some more details to help me understand the issue?